analytics.go

```diff
@@ -239,10 +241,21 @@ func (c *Client) Close() error {
 	return nil
 }
 
+func (c *Client) sendAsync(msgs []interface{}) {
+	c.wg.Add(1)
+	go func(wg *sync.WaitGroup) {
```

Comment:
You don't need to pass it here, since it's already capturing the `c` variable, you can just access `c.wg` in this closure.